### PR TITLE
NDLAR Dead material change

### PR DESCRIPTION
**Pull request by zhulcher**
While doing an analysis looking at the effect of dead material in NDlar on reconstruction efficiencies, I noticed an approximately percent difference in the fraction of dead volume in NDlar between the CAD geometry which I have measurements from and the GDML, which I think might be relevant for my study. I believe i have made changes that fix it, but I would appreciate a second look! Specifically the gap dimensions and construction of the connection to the backplate don't currently match what I see in the CAD.

Thanks in advance,

Zach

### CODE REVIEW

**Comment by ast0815** [comment]
I don't know anything about the NDLAr geometry. Do you have a suggestion who should review the geometry changes?

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,8 @@ dunendggd.egg-info
 *~
 *#
 *.bak
+*.xml
+*plotting.py
```
Comment (ast0815) [inline .gitignore:10]: What is this `*plotting.py`? Should this really be ignored in the main repository? Same with the xml files. Are they a product of the workflow in the repository?

File: duneggd/Config/ArgonCube/ArgonCubeDetector.cfg [hunk 2]
```diff
@@ -1,114 +1,81 @@
...
-N_Pixel                 = 0
-#N_Pixel                 = 70
-N_Asic                  = 10
+class = duneggd.ArgonCube.PixelPlane.PixelPlaneBuilder
+subbuilders = []
+PCB_dimension = {'dx':Q('3.175mm')/2,'dy':Q('299.599mm')/2,'dz':Q('477.85999999999996mm')/2}
+Pixel_dimension = {'dx':Q('0.051mm')/2,'dy':Q('4.000mm')/2,'dz':Q('4.000mm')/2}
+Asic_dimension = {'dx':Q('1.422mm')/2,'dy':Q('16.0mm')/2,'dz':Q('16.0mm')/2}
+N_Pixel = 0
+N_Asic = 10
```
Comment (ast0815) [inline duneggd/Config/ArgonCube/ArgonCubeDetector.cfg:18]: A lot of changes are just you removing spaces. Could you keep with the established config formatting (i.e. aligned equal signs) and only change the lines where you actually change something? Always better to keep things consistent. And you won't get git-blamed for lines where you did not actually change any parameters. ;)

Removing commented out configuration lines is totally fine though. Keeping comments of how things used to be before a change is a bit useless when we have the complete commit history.

**Review by ast0815 — CHANGES_REQUESTED** [review]

**Comment by zhulcher** [comment]
That should be a bit better now! I know Andrew Mastbaum edited part of the ND module geometry a while back? Otherwise, not the foggiest

File: duneggd/ArgonCube/NDBucket.py [hunk 3]
```diff
@@ -169,5 +169,4 @@ def construct(self,geom):
                                                             pos=FieldcageGap_pos,
                                                             copynumber=2*ii*jj)
 
-                ArgonColumn_lv.placements.append(FieldcageGap_pla.name)
-
+                ArgonColumn_lv.placements.append(FieldcageGap_pla.name)
```
Comment (ast0815) [inline duneggd/ArgonCube/NDBucket.py:172]: Your editor seems to not add newlines at the end of files. Could you fix this as well?

**Comment by ast0815** [comment]
Looks much better. Could you add a line to the CHANGELOG.md about what you changed?

I will ask around who could do the review.

**Comment by ast0815** [comment]
Chris wrote me an e-mail, saying the changes look fine to him, and that he trusts you @zhulcher, that the changes are warranted.

Could you address the last few issues I flagged up, so we can merge this PR?

**Review by ast0815 — APPROVED** [review]

**Pull request merged**